editor/src/components/common/shared-strategies/convert-to-flex-strategy.spec.browser2.tsx

```diff
@@ -991,7 +993,564 @@ describe('Smart Convert To Flex if Fragment Children', () => {
     await expectSingleUndoStep(editor, () => clickOnPlusButton(editor))
```

Comment:
Heads up that depending on the order we merge in, this test will clash with #3396 now that it does something. If I merge first I'll fix this